docs/documentation/cn/sql-reference/sql-statements/Data Definition/CREATE TABLE.md

```diff
@@ -18,136 +18,163 @@ under the License.
-            "database" = "database_name",
-            "table" = "table_name"
-            )
-        
-        注意：
-            "table" 条目中的 "table_name" 是 mysql 中的真实表名。
-            而 CREATE TABLE 语句中的 table_name 是该 mysql 表在 Palo 中的名字，可以不同。
-            
-            在 Palo 创建 mysql 表的目的是可以通过 Palo 访问 mysql 数据库。
-            而 Palo 本身并不维护、存储任何 mysql 数据。
-        2) 如果是 broker，表示表的访问需要通过指定的broker, 需要在 properties 提供以下信息：
-            PROPERTIES (
-            "broker_name" = "broker_name",
-            "path" = "file_path1[,file_path2]",
-            "column_separator" = "value_separator"
-            "line_delimiter" = "value_delimiter"
-            )
-            另外还需要提供Broker需要的Property信息，通过BROKER PROPERTIES来传递，例如HDFS需要传入
-            BROKER PROPERTIES(
-                "username" = "name", 
-                "password" = "password"
-            )
-            这个根据不同的Broker类型，需要传入的内容也不相同
-        注意：
-            "path" 中如果有多个文件，用逗号[,]分割。如果文件名中包含逗号，那么使用 %2c 来替代。如果文件名中包含 %，使用 %25 代替
-            现在文件内容格式支持CSV，支持GZ，BZ2，LZ4，LZO(LZOP) 压缩格式。
+    `col_name col_type [agg_type] [NULL | NOT NULL] [DEFAULT "default_value"]`
+
+    说明：
+    col_name：列名称
+    col_type：列类型
+
+    ```
+        TINYINT（1字节）
+            范围：-2^7 + 1 ~ 2^7 - 1
+        SMALLINT（2字节）
+            范围：-2^15 + 1 ~ 2^15 - 1
+        INT（4字节）
+            范围：-2^31 + 1 ~ 2^31 - 1
+        BIGINT（8字节）
+            范围：-2^63 + 1 ~ 2^63 - 1
+        LARGEINT（16字节）
+            范围：-2^127 + 1 ~ 2^127 - 1
+        FLOAT（4字节）
+            支持科学计数法
+        DOUBLE（12字节）
+            支持科学计数法
+        DECIMAL[(precision, scale)] (16字节)
+            保证精度的小数类型。默认是 DECIMAL(10, 0)
+            precision: 1 ~ 27
+            scale: 0 ~ 9
+            其中整数部分为 1 ~ 18
+            不支持科学计数法
+        DATE（3字节）
+            范围：1900-01-01 ~ 9999-12-31
+        DATETIME（8字节）
+            范围：1900-01-01 00:00:00 ~ 9999-12-31 23:59:59
+        CHAR[(length)]
+            定长字符串。长度范围：1 ~ 255。默认为1
+        VARCHAR[(length)]
+            变长字符串。长度范围：1 ~ 65533
+        HLL (1~16385个字节)
+            hll列类型，不需要指定长度和默认值、长度根据数据的聚合
+            程度系统内控制，并且HLL列只能通过配套的hll_union_agg、Hll_cardinality、hll_hash进行查询或使用
+        BITMAP
+            bitmap 列类型，不需要指定长度和默认值
+            BITMAP 列只能通过配套的 BITMAP_UNION、BITMAP_COUNT、TO_BITMAP 进行查询或使用
+    ```
+
+    agg_type：聚合类型，如果不指定，则该列为 key 列。否则，该列为 value 列
+       * SUM、MAX、MIN、REPLACE
+       * HLL_UNION(仅用于HLL列，为HLL独有的聚合方式)、
+       * BITMAP_UNION(仅用于 BITMAP 列，为 BITMAP 独有的聚合方式)、
+       * REPLACE_IF_NOT_NULL：这个聚合类型的含义是当且仅当新导入数据是非NULL值时会发生替换行为，如果新导入的数据是NULL，那么Doris仍然会保留原值。注意：如果用在建表时REPLACE_IF_NOT_NULL列指定了NOT NULL，那么Doris仍然会将其转化NULL，不会向用户报错。用户可以借助这个类型完成部分列导入的功能。
+       * 该类型只对聚合模型(key_desc的type为AGGREGATE KEY)有用，其它模型不需要指这个。
+
+    是否允许为NULL: 默认不允许为 NULL。NULL 值在导入数据中用 \N 来表示
+
+    注意：
+        BITMAP_UNION聚合类型列在导入时的原始数据类型必须是TINYINT,SMALLINT,INT。
+
+2. index_definition
+    语法：
+        `INDEX index_name [USING BITMAP] (col_name[, col_name, ...]) COMMENT 'xxxxxx'`
```

Comment:
I think should keep it compatible with create index syntax
```
INDEX index_name (col_name[, col_name, ...])  [USING BITMAP] COMMENT 'xxxxxx'
```

Reply:
done